pkg/apicapi/apicapi.go

```diff
@@ -811,11 +818,11 @@ func (conn *ApicConnection) subscribe(value string, sub *subscription) bool {
 		kind = "class"
 	}
 
-        refresh_interval := ""
-        if conn.RefreshInterval != 0 {
-                refresh_interval = fmt.Sprintf("refresh-timeout=%s&",
-                        conn.RefreshInterval)
-        }
+	refresh_interval := ""
+	if conn.RefreshInterval != 0 {
+		refresh_interval = fmt.Sprintf("refresh-timeout=%s&",
+			conn.RefreshInterval)
+	}
```

Comment:
I believe there is no change needed here, so this should not show up in the diff. Did something change with the formatting?

Reply:
It was spaces before but gofmt convert them to tabs.